lib/src/emoji_lists.dart

```diff
@@ -1189,6 +1189,7 @@ final Map<String, String> foods = Map.fromIterables([
   'Glass of Milk',
   'Hot Beverage',
   'Teacup Without Handle',
+  'Mate Tee',
```

Comment:
Change to official Unicode Name
```suggestion
  'Mate Drink',
```